docs/en/annotations.rst

```diff
@@ -257,7 +257,7 @@ PHP Imports
 
 By default the annotation reader parses the use-statement of a php file
 to gain access to the import rules and register them for the annotation
-processing. Only if you are using PHP Imports you can validate the
+processing. Only if you are using PHP Imports can you validate the
```

Comment:
This change looks weird to me. `can you validate` looks like a question to me, but this is not a question.

Reply:
This is fine from a grammar perspective 👍